java/display-kml/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,6 @@
     package="com.esri.arcgisruntime.displaykml">
 
     <uses-permission android:name="android.permission.INTERNET" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
We need to leave this in the old java one, because it's still using the old pattern of reading from an external folder to the sample

https://github.com/Esri/arcgis-runtime-samples-android/blob/kotlin_display_kml/java/display-kml/src/main/java/com/esri/arcgisruntime/displaykml/MainActivity.java#L128

Reply:
whoops! must have got my windows confused